### PR TITLE
describe.limits.leverage added to bittrex, bybit, coinbase, kucoin, huobi, okex

### PR DESCRIPTION
**Pull request by samgermain**
Also added superLeverage to huobi
I also removed `describe.limits.leverage.min` because 
 - most exchanges don't give it
 - Those that do (bybit and gateio) give a value of 1 every single time
 - I thought it could lead to confusion because some margin markets give a max leverage value (e.g. 10) and then you decide your actual leverage based on how much collateral you decide to spend, but other exchanges, like most futures exchanges, multiply your collateral by the value that your leverage is set to 
 
 ----------------------
 
 It looks like Phemex might have a leverage structure similar to binance (with leverage brackets) but they provide these brackets in the `fetchMarkets` response. 
 
 ```
         info: {                   symbol:   "ATOMUSD",
                           displaySymbol:   "ATOM / USD",
                            ...
                             tipOrderQty:   "7500",
                                   steps:   "500K",
                              riskLimits: [ {               limit: "500000",          // This is the part that looks like leverage_brackets
                                                    initialMargin: "5.0%",
                                                  initialMarginEr: "5000000",
                                                maintenanceMargin: "1.0%",
                                              maintenanceMarginEr: "1000000"  },
                                            {               limit: "1000000",
                                                    initialMargin: "5.5%",
                                                  initialMarginEr: "5500000",
                                                maintenanceMargin: "1.5%",
                                              maintenanceMarginEr: "1500000"  },
                                            {               limit: "1500000",
                                                    initialMargin: "6.0%",
                                                  initialMarginEr: "6000000",
                                                maintenanceMargin: "2.0%",
                                              maintenanceMarginEr: "2000000"  },
                                            {               limit: "2000000",
                                                    initialMargin: "6.5%",
                                                  initialMarginEr: "6500000",
                                                maintenanceMargin: "2.5%",
                                              maintenanceMarginEr: "2500000"  },
                                            ...
                                            {               limit: "5000000",
                                                    initialMargin: "9.5%",
                                                  initialMarginEr: "9500000",
                                                maintenanceMargin: "5.5%",
                                              maintenanceMarginEr: "5500000"  }  ] },
```
 
 I could create a unified response by adding binance and phemex's leverage brackets to `limits:leverage` but that would require another api call on binance futures to `fapiPrivateGetLeverageBracket`
`dapiPrivateV2GetLeverageBracket`. 

This might be a good idea because 
- it's looking like every other exchange provides this information when you fetch the markets
- Anyone trading on futures would probably want this information and need to make both calls anyway (unless maybe they're always trading below 20x)
- It should probably be handled in the same singleton type manner that the markets are handled, and refreshed accordingly when the markets are refreshed

Let me know what you think about this part

### CODE REVIEW

**Comment by frosty00** [comment]
The minimum leverage will probably always be 1, cause it doesn't really make sense to be lower and it doesn't make sense to be higher

**Comment by samgermain** [comment]
> The minimum leverage will probably always be 1, cause it doesn't really make sense to be lower and it doesn't make sense to be higher

I figured that, but some exchanges explicitly provided it

File: js/coinbase.js [hunk 1]
```diff
@@ -572,6 +572,9 @@ module.exports = class coinbase extends Exchange {
                                 'min': this.safeNumber (quoteCurrency, 'min_size'),
                                 'max': undefined,
                             },
+                            'leverage': {
+                                'max': 1,
```
Comment (frosty00) [inline js/coinbase.js:576]: i don't think we should define this value here...

Comment (samgermain) [reply js/coinbase.js:576]: why

**Pull request merged**